go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/moby/term v0.5.0
 	github.com/replicate/go v0.0.0-20250205165008-b772d7cd506b
+	github.com/sabhiram/go-gitignore v0.0.0-20210923224102-525f6e181f06
```

Comment:
Any alternatives out there that haven't been abandoned? 😬

Reply:
I can't find any, but let me know if you can suggest an alternative

Reply:
🤷🏼